apps/ledger-live-mobile/src/screens/SyncOnboarding/EarlySecurityCheckBody.tsx

```diff
@@ -153,9 +154,14 @@ const EarlySecurityCheckBody: React.FC<Props> = ({
       }
 
       secondaryBottomCta = (
-        <Link size="large" onPress={onSkipFirmwareUpdate}>
+        <OnePressCta
+          PressableComponent={({ pending, ...props }) => (
+            <Link size="large" disabled={pending} {...props} />
```

Comment:
you could also set the `Icon` prop of link to an `InfiniteLoader` because just disabling it could be confusing

Reply:
The definition of done isn't very clear in [LIVE-8424](https://ledgerhq.atlassian.net/browse/LIVE-8424) but it's mentioned to "make it pressed" to reflect the loading state. 
I'll make this proposal to the QA

Reply:
Yeah IMO a loader is better because "make it pressed" might look like the UI has frozen